### ISSUE TITLE
How do you get the final predicted maskiou?

### ISSUE THREAD

**Issue by JoyHuYY1412** [zjhuang22/maskscoring_rcnn#49]
In your code, the maskiou value is predicted by the network and you use l2-loss.
But the observation of GT maskiou is [0,1], so is it better to do ReLu after you calculated the iou using the Linear layer?

**Comment by zjhuang22** [comment]
Thank you for your advice. The use of Relu has little effect.

**Comment by JoyHuYY1412** [comment]
> Thank you for your advice. The use of Relu has little effect.

Yes, but I think it is weird if the predict maskiou is negative and then you multiply it with a non-negative confidence score.

**Comment by zjhuang22** [comment]
In the training stage, the ground truth maskiou is large than 0 and most of them larger than 0.5, so the predicted maskiou will be larger than 0 in the testing stage.

**Comment by Elsanna** [comment]
我将maskiou_head中的pred_maskiou打印出来，发现有两列，第一列的数字有正有负，第二列的数字都是正的，想请问这两列分别代表什么含义呢？
